docs/release-notes/2157-contrastive-learning.txt

```diff
@@ -0,0 +1,7 @@
+:orphan:
+
+**New Features**
+
+-  Add new example for contrastive learning with support for
+   [SIMCLR](https://arxiv.org/abs/2002.05709) and [supervised
```

Comment:
This isn't reStructuredText notation for links. They should be something like:
-  \`SIMCLR \<https://arxiv.org/abs/2002.05709>`_
-  \`supervised contrastive learning \<https://arxiv.org/abs/2004.11362>`_